server/views/admin-list-articles.hbs

```diff
@@ -1,8 +1,6 @@
-
-    <a class="addNewArticle" href="articles/add">
+<a class="addNewArticle" href="articles/add">
```

Comment:
this is invalid HTML - having a button inside an `A` tag. Just use bootstrap Button tags http://getbootstrap.com/docs/4.0/components/buttons/#button-tags